typings/index.d.ts

```diff
@@ -1008,6 +1008,60 @@ export interface ComposerConstructor {
 
 export const Telegraf: TelegrafConstructor;
 
+export interface LaunchPollingOptions {
+  /**
+   * Poll timeout in seconds
+   */
+  timeout?: number
+
+  /**
+   * Limits the number of updates to be retrieved
+   */
+  limit?: number
+
+  /**
+   * List the types of updates you want your bot to receive
+   */
+  allowedUpdates?: tt.UpdateType[] | tt.UpdateType | null
+
+  /**
+   * Polling stop callback
+   */
+  stopCallback?: () => void | null
```

Comment:
`null` because the documentation explicitly mentions that the default value is null